src/devices/Tm16xx/samples/Sample.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Iot.Device.Tm16xx;
```

Comment:
you are still missing the csproj associated to the sample. Also, by convention, we name them "Program.cs"